### PR TITLE
added note about removing repititive code or content in the introduction Issue 1108

### PR DESCRIPTION
**Pull request by alexkarasik**
Here is the rawgit link: http://rawgit.com/alexkarasik/alexkarasik.github.io/lessismore/#!pages/githubissues.md

Not sure if it is necessary to add it as a note. I'll just include it as a sentence in the introduction if anyone thinks that is more appropriate. 

[x] Check for issue number in pull request title
[x] Are there any unneeded files in the pull request?
[x] Did they make a branch for their patch?
[x] Does the pull request actually fix the issue?
[x] Check the pull request on rawgit, does it display without any errors?
[x] Is there any merge conflicts?
[x] Make sure that people use their GitHub accounts when making commits through git

### CODE REVIEW

File: pages/githubissues.md [hunk 1]
```diff
@@ -7,8 +7,16 @@
 * Work with admins to make adjustments if necessary, and merge your branch into the upstream
 
 ## Introduction
+
+On GitHub, each repository has a section where issues can be added, discussed, and fixed as a means of categorizing and addressing problems we find. Issues are often opened by our team to fix software bugs on the BeLL and in our code.
+
+**Note**: Issues are not just about adding to the wiki, but can also involve removing repetitive or unnecessary code or content. 
+
+Below, you'll find an example of an issue where I brought up a problem, researched an answer, and then fixed the issue.
```
Comment (kylemathias) [inline pages/githubissues.md]: What issue are you referring to? this sentence sounds like there is an issue you are going to link to. Try changing it around or remove it.

Comment (alexkarasik) [inline pages/githubissues.md]: The only part I added was the note, saying that removing repetitive or unnecessary code or content could be a valid issue. I thought it belonged as a note in the introduction when describing what issues are or could be. Do you think I should put this note somewhere else?

Comment (kylemathias) [inline pages/githubissues.md]: no, i think the note is really well placed. it looks like you added the other sentence because it was moved.

Comment (alexkarasik) [inline pages/githubissues.md]: I think I just added a space between those lines for appearance sake. That sentence was already in there.

**Pull request merged**